src/js/tech/ChromecastTech.js

```diff
@@ -165,6 +167,9 @@ ChromecastTech = {
       mediaInfo.metadata.metadataType = chrome.cast.media.MetadataType.GENERIC;
       mediaInfo.metadata.title = title;
       mediaInfo.metadata.subtitle = subtitle;
+      if (customData) {
```

Comment:
It appears this still won't [allow for a falsey value to be passed via `customData`](https://github.com/silvermine/videojs-chromecast/pull/20#discussion_r184977990). Was there a particular reason you decided to go this route?

Reply:
IIRC, I believe it's possible that the `mediaInfo` object could already have an existing `customData` object, and I didn't want to overwrite any existing values if it wasn't set in the plugin's config on the page. (I remember seeing at one point that, even though there didn't appear to be a `customData` object set in `mediaInfo` when logged to the console locally, an object had indeed been passed to the receiver and logged to the remote console.)

Reply:
@onebytegone does this still need to be refactored, or can my previous comment justify merging this PR?